setup.py

```diff
@@ -146,7 +146,7 @@ def write_version(filename=os.path.join(*['airflow',
 postgres = ['psycopg2-binary>=2.7.4']
 ssh = ['paramiko>=2.1.1', 'pysftp>=0.2.9']
 salesforce = ['simple-salesforce>=0.72']
-s3 = ['boto3>=1.0.0']
+s3 = ['boto3>=1.4.0']
```

Comment:
Are you sure it it is safe to bump this version?

Reply:
Thanks for the review @Fokko! 
In addition to confirming that all tests succeeded, I checked [Boto3's upgrading notes](https://boto3.readthedocs.io/en/latest/guide/upgrading.html) and confirmed that all classes mentioned there are not used (at least directly) in the current codebase. If you have any further concerns, I'll investigate it.